docs/workers.rst

```diff
@@ -115,6 +115,18 @@ To manipulate a specific worker, you pass the -w option to synctl::
     synctl -w $CONFIG/workers/synchrotron.yaml restart
 
 
+After setting up your workers, you'll need to create a worker configuration for
+the main synapse process. That worker configuration should look like this:::
+
+    worker_app: synapse.app.homeserver
+    daemonize: true
+
+Be sure to keep this particular configuration limited as synapse may refuse to
+start if the regular ``worker_*`` options are given. The ``homeserver.yaml``
+configuration will be used to set up the main synapse process.
```

Comment:
isn't homeserver.yaml used for all of the processes?

Reply:
It is, however the main worker is different in that it refuses to work with the standard worker_* options. I wanted to add a reminder that the homeserver.yaml is used instead.

Reply:
I think your phrasing is a bit misleading then.